shaptools/netweaver.py

```diff
@@ -416,3 +416,20 @@ def stop(self, wait=15, delay=0, exception=True, **kwargs):
         if exception and result.returncode:
             raise NetweaverError('Error running sapcontrol command: {}'.format(result.cmd))
         return result
+
+    def extract_sap_installer(
```

Comment:
I don't think we need special function for each SAR file. The extraction is a genetic process. For me it is fine to raise an generic extraction error pointing to the file.